docs/en/docs/kant/user-guide/device/create-device.md

```diff
@@ -16,19 +16,20 @@ Follow these steps to create an end device:
 
 1. Navigate to the edge unit details page and select __Edge Resources__ -> __End Devices__ from the left menu.
 
-    ![End Devices](../images/create-device-01.png)
-
 2. Click the __Create Device__ button located at the top right corner of the end device list.
 
-    ![Create Device](../images/create-device-02.png)
+    <!--add image later-->
 
 3. Enter the basic information.
 
     - Device Name: Use a combination of lowercase letters, numbers, hyphens (-), and dots (.). Avoid consecutive separators and ensure the name starts and ends with a letter or number. The maximum character limit is 253.
```

Comment:
```suggestion
    - Device Name: Use a combination of lowercase letters, numbers, hyphens (-), and periods (.). Avoid consecutive symbols and ensure the name starts and ends with a letter or number. It can have a maximum of 253 characters.
```